test/server/queries_test.js

```diff
@@ -10,4 +10,16 @@ describe('Queries', function(){
     })
   })
 
+  describe('getPrrrsPerWeek', () =>{
```

Comment:
yay tests! I actually never wrote tests for the metrics page I added. I'll add a task for this. Feel free to modify this to test the `metricsForWeek` query